results/docs/Control_benchmarks_analysis.md

```diff
@@ -0,0 +1,37 @@
+# Control Benchmarks 
+
+The suite of control conditional benchmarks which is a part of the [university of wisconsin micro-benchmarks suite](https://github.com/VerticalResearchGroup/microbench) can be used to try and understand the difference in performance between the intel skylake processor and the branch prediction unit in gem5.
+
+Control Conditional Benchmarks:
+
+- CCa : completely baised branches
+
+- CCe : easy to predict -- branch pattern (10101010...)
+
+- CCm : heavly baised branches
+
+- CCh_st : impossible to predict with a store operation
+
+
+**Understanding the role of Branch Miss-Predictions**
+
+Inorder to understand the role of branch predictors, we take a  closer look into it by ploting the conditional branch misprediction rate while using LTAGE branchmispredictor (which is the best branchpredictor)
+
+The misprediction rate is almost same as the intel pcm for both UnCalibrated and Calibrated CPU. 
+
+While increasing the iteration to '500K', the misprediction comes down for gem5 while using LTAGE because LTAGE has a bigger history table, therefore gets better with increase in iterations.
```

Comment:
```suggestion
Then increasing the iteration to '500K', the misprediction rate reduces for gem5 while using LTAGE because LTAGE has a larger history table.
This large history table takes time to warm up and, thus, we see increased prediction rates as we increase the number of iterations.
```